From fbe03f950fb3016eec4a270fbe4a4c42e4ec60e1 Mon Sep 17 00:00:00 2001
From: Kumar Gala <galak@kernel.crashing.org>
Date: Fri, 20 May 2011 00:00:21 -0500
Subject: [PATCH 022/233] powerpc/qe: Limit QE support to ppc32

Extracted from vendor drop QorIQ-DPAA-SDK-V1-20110609-systembuilder.iso
0089-powerpc-qe-Limit-QE-support-to-ppc32.patch

Only 32-bit SoCs have a QUICC Engine so limit the config option to PPC32.

Signed-off-by: Kumar Gala <galak@kernel.crashing.org>
---
 arch/powerpc/platforms/Kconfig |    2 +-
 1 files changed, 1 insertions(+), 1 deletions(-)

diff --git a/arch/powerpc/platforms/Kconfig b/arch/powerpc/platforms/Kconfig
index 5bb7f62..7f19660 100644
--- a/arch/powerpc/platforms/Kconfig
+++ b/arch/powerpc/platforms/Kconfig
@@ -248,7 +248,7 @@ config TAU_AVERAGE
 
 config QUICC_ENGINE
 	bool "Freescale QUICC Engine (QE) Support"
-	depends on FSL_SOC
+	depends on FSL_SOC && PPC32
 	select PPC_LIB_RHEAP
 	select CRC32
 	help
-- 
1.7.0.4

